src/Core/Framework/Adapter/Cache/Http/CacheResponseSubscriber.php

```diff
@@ -230,9 +238,14 @@ private function buildCacheHash(Request $request, SalesChannelContext $context):
             HttpCacheCookieEvent::VERSION_ID => $context->getVersionId(),
             HttpCacheCookieEvent::CURRENCY_ID => $context->getCurrencyId(),
             HttpCacheCookieEvent::TAX_STATE => $context->getTaxState(),
-            HttpCacheCookieEvent::LOGGED_IN_STATE => $context->getCustomer() ? 'logged-in' : 'not-logged-in',
```

Comment:
I think this might still be required, as for example the account menu (present on each page), is different depending if one is logged in or not.

Reply:
you are absolutely right

i first thought that account menu was fetched via ajax together with the cart info, but it's not, so we keep it for now 👍